Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.22.1-alpine
+
+COPY . .
```

Comment:
If you use `COPY`, then how will hot-reloading work? Or is this only for the packaged final runtime app, and you are going to create a different Dockerfile for development purposes - which will incorporate volume mounting for hot reloading?